src/main/java/com/moriwaki/java10thtopic/controller/FishController.java

```diff
@@ -81,4 +82,11 @@ public ResponseEntity<String> update(@RequestBody FishUpdateRequest fishUpdateRe
         return ResponseEntity.ok("fish date updated");
     }
 
+    //削除処理
+    @DeleteMapping("/fishes/{id}")
+    public ResponseEntity<String> delete(@PathVariable("id") int id){
+        fishService.delete(id);
+        return ResponseEntity.ok("fish date deleted");
```

Comment:
日付になってますね。
気が付いていなかったですが、POSTとPATCHもです🙏
```suggestion
        return ResponseEntity.ok("fish data deleted");
```